network-costs-configuration.md

```diff
@@ -33,6 +33,32 @@ networkCosts:
   enabled: true
 ```
 
+## Additional configuration
+
+You can view a list of common config options [here](https://github.com/kubecost/cost-analyzer-helm-chart/blob/700cfa306c8e78bc9a1039b584769b9a0e0757d0/cost-analyzer/values.yaml#L573).
+
+### Prometheus:
+
+- If using the included Prometheus instance, the scrape is automatically configured.
```

Comment:
In this context, is this referring to Kubecost-bundled Prometheus?

Reply:
Correct! This is referring to Kubecost's bundled Prometheus

Reply:
No worries! I agree that we should standardize on how we refer to "Kubecost-bundled Prom". I'll make the change later today. Thanks!

Reply:
@bstuder99 Fixed!